src/core/operations/AESEncrypt.mjs

```diff
@@ -98,7 +106,8 @@ class AESEncrypt extends Operation {
     run(input, args) {
         const key = Utils.convertToByteString(args[0].string, args[0].option),
             iv = Utils.convertToByteString(args[1].string, args[1].option),
-            mode = args[2],
+            mode = args[2].substring(0, 3),
```

Comment:
This seems brittle, could you use a `.split('/')` instead, in case the first string is not three characters?